tests/system/python/api/test_configuration.py

```diff
@@ -127,6 +127,14 @@ def test_default(self, fledge_url, reset_and_start_fledge, wait_time, storage_pl
                                      "displayName": "Password Policy",
                                      "children": [
 
+                                     ]
+                                 },
+                                 {
+                                     "key": "firewall",
```

Comment:
this does not match the given default config; Are we not testing against that object?

Reply:
This test is only to check the default heirarchy tree for category without detailed config